portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+public class Comment {
+    private String content;
+    private long timestamp;  // System.currentTimeMillis()

```

Comment:
Nit: inline comment not needed.

Reply:
Noted 👍

Reply:
I feel that it would make clear to readers in what format the timestamp is though, why do you think it's not needed?

Reply:
We normally indicate the unit or the standard for vairables instead of the code on how to get them.
E.g. `Epoch/Unix timestamp` would be a better comment then `System.currentTimeMillis()`